proto/storage/report_snapshot.proto

```diff
@@ -25,12 +25,16 @@ message ReportSnapshot {
   ReportType type = 5;
   oneof filter {
     VulnerabilityReportFilters vuln_report_filters = 6;
+    ViewBasedVulnerabilityReportFilters view_based_vuln_report_filters = 12;
   }
   CollectionSnapshot collection = 7;
   storage.Schedule schedule = 8;
   ReportStatus report_status = 9;
   repeated NotifierSnapshot notifiers = 10;
   SlimUser requester = 11;
+  //fields related to ondemand reports
+  string view_based_report_request_name = 13;
```

Comment:
what separates this from `name` at line 23?  Will that field still be populated?

Reply:
yes we can reuse name field for request name